bin/improver-uv-index

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# (C) British Crown Copyright 2017-2018 Met Office.
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# * Redistributions of source code must retain the above copyright notice, this
+#   list of conditions and the following disclaimer.
+#
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#
+# * Neither the name of the copyright holder nor the names of its
+#   contributors may be used to endorse or promote products derived from
+#   this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+"""Script to run the UV index plugin."""
+
+from improver.argparser import ArgParser
+from improver.utilities.load import load_cube
+from improver.utilities.save import save_netcdf
+from improver.uv_index import calculate_uv_index
+
+
+def main():
+    """ Calculate the UV index using the data
+    in the input cubes."""
+    parser = ArgParser(
+        description="Calculates the UV index.")
+    parser.add_argument("radiation_flux_upward",
+                        metavar="RADIATION_FLUX_DOWNWARD",
+                        help="Path to a NetCDF file of radiation flux "
+                        "in uv downward at surface.")
+    parser.add_argument("radiation_flux_downward",
+                        metavar="RADIATION_FLUX_UPWARD",
+                        help="Path to a NetCDF file of radiation flux "
+                        "in uv upward at surface.")
+    parser.add_argument("output_filepath", metavar="OUTPUT_FILE",
```

Comment:
I know you have made some changes to set the scale_factor to 3.6 within the plugin, but there was a whole debate about the scale factor when I  wrote the plugin - in the end we decided to leave it as a scale factor that needed to be set by the user.  I think we were thinking that it would be set at the CLI rather than within the plugin. A note saying that this shouldn't be changed unless there is very good reason would be a good idea!
 
(more information on the discussions we had can be found here: https://github.com/metoppv/improver/pull/747)

Reply:
I think we need to set 3.6 in the plugin as it isn't an arbitrary value but has been tuned to be in keeping with the WMO definition of the UV Index

[https://en.wikipedia.org/wiki/Ultraviolet_index](https://en.wikipedia.org/wiki/Ultraviolet_index)

so it isn't something we want the user to be changing. I've added it in as an optional configuration keyword with a default value of 3.6 in the CLI but I've added a warning that it shouldn't be changed unless there is a scientific reason for doing so.